openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -12,6 +12,7 @@
 from django.core.validators import ValidationError, validate_email
 from django.utils.translation import override as override_language
 from django.utils.translation import gettext as _
+from eventtracking import tracker
```

Comment:
what is this tracker? where does it emit events? why are we not using segment.track?

Reply:
`segment.track` requires `user_id` but in our case user is not authenticated yet.
Reference: https://github.com/openedx/edx-platform/blob/master/common/djangoapps/track/segment.py#L19